background.js

```diff
@@ -46,7 +46,7 @@ function check() {
                 })
         }
     })
-    window.setTimeout(check, 10000);
+    window.setTimeout(check, 60000);
```

Comment:
From a UX point of view, not sure I like this one. It will lead to a big gap while you're in a review session.
I think 1 call every 10 seconds is very reasonable, but I'm open to discussion.

Reply:
This is fair enough. I don't like that the API is checked every 10 seconds, that's a total of 8640 requests a day, which is quite excessive since WK reviews are only available on the hour. I'm happy to revert this change for now, but I would like to see if the extension could check when you finish your reviews on WK and force-check then, as well as every hour. Let me know your thoughts, I'm sure it's possible but I'll have to look into it.

Reply:
Totally agree with you here in principle. Technically 10 sec is still within the [api specs](https://docs.api.wanikani.com/20170710/#rate-limit), but still not a smart nor necessary use of server resources. Ideally, the extension should check whether the active tab is either wanikani.com/lesson or wanikani.com/review. If not, as you say, there's no need to check so often.
Shall we write this down as future development and discard the change for now, to maintain the current UX?